include/nighthawk/client/options.h

```diff
@@ -9,6 +9,8 @@
 #include "envoy/config/cluster/v3/cluster.pb.h"
 #include "envoy/config/core/v3/base.pb.h"
 
+#include "nighthawk/common/termination_predicate.h"
```

Comment:
A little surprised to see this include with no other corresponding changes.  Why was this suddenly necessary?

Reply:
this is an include-what-you-use fix; the need for which became apparent when isolating the compilation unit. Apparently, before doing that, the include leaded into scope via another include (the original mock_options.h) in a test which needed it.